shapeworks_cloud/core/rest.py

```diff
@@ -78,8 +78,16 @@ def set_thumbnail(self, request, **kwargs):
     def subset(self, request, **kwargs):
         dataset = self.get_object()
         form_data = request.data
+        selected = form_data.get('selected')
+        name = form_data.get('name') or dataset.name + '_subset'
+        if len(selected) < 1:
+            return Response(
```

Comment:
Is there a way to get to this code path?  When I deselect the shape or all of the subjects, the create subset button is disabled (which is good).

Reply:
I suppose not from the UI, but since we have an extensible python client (to which we may add a subset function), I figured it was best to protect against that case anyway.